.github/workflows/cifuzz.yaml

```diff
@@ -0,0 +1,26 @@
+name: CIFuzz
+on:
+  pull_request:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+jobs:
+  Fuzzing:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Restore Go cache
+      uses: actions/cache@v1
+      with:
+        path: /home/runner/work/_temp/_github_home/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+    - name: Fetch CRDs
+      run: make test_deps
```

Comment:
This must happen within the `build.sh` script. To support the oss-fuzz integration that script must be self sufficient to setup all dependencies required to build the application.